src/main/java/co/com/perficient/project3/utils/constant/StadiumConstants.java

```diff
@@ -0,0 +1,9 @@
+package co.com.perficient.project3.utils.constant;
+
+import lombok.AccessLevel;
+import lombok.NoArgsConstructor;
+
+@NoArgsConstructor(access = AccessLevel.PRIVATE)
+public final class StadiumConstants {
+    public static final String STADIUM_ENDPOINT = "/api/stadium";
```

Comment:
It isn't too convenient as "API_STADIUM" and why did you use "API" always?
 It's really useful?